rebabel_format/converters/nlp_part_of_speech.py

```diff
@@ -31,19 +32,24 @@ def process_line(self, line: str):
         """Process one word, part of speech pair at a time.
 
         Positional arguments:
-        line -- a word and its part of speech separated by the / delimiter (ex: jumped↑VERB)
+        line -- a word and its part of speech separated by a delimiter (ex: jumped/VERB)
         """
         split_line = line.strip().split(" ")
+        delimiter = "/"
```

Comment:
this should be a class attribute `delimiter = Parameter(default='/', ...)` so it can show up in the documentation